README.md

```diff
@@ -73,112 +72,113 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.62 - August 2022 Update
+### 0.63 - September 2022 Update
 
-In this release, we focused on releasing three new PowerToys.
+In this release, we focused on stability and improvement.
 
 **Highlights**
 
-- New utility: Screen Ruler is a quick and easy way to measure pixels on your screen.
-- New utility: Quick Accent is an easy way to write letters with accents. Thanks [@damienleroy](https://github.com/damienleroy)!
-- New utility: Text Extractor works like Snipping Tool, but copies the text out of the selected region using OCR and puts it on the clipboard. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
-- PowerToy Run ships with a new Plugin letting you search in past query results. Thanks [@jefflord](https://github.com/jefflord)!
+- QuickAccent contains a new setting to select a language. This should reduce the number of accented characters a user needs to pick from. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Reduced installer and installed sizes by sharing the Windows App SDK, VC++ redistributable and PowerToys Interop runtime files between utilities.
 
 ### Known issues
+
 - The Text Extractor utility [fails to recognize text in some cases on ARM64 devices running Windows 10](https://github.com/microsoft/PowerToys/issues/20278).
 - After installing PowerToys, [the new Windows 11 context menu entries for PowerRename and Image Resizer might not appear before a system restart](https://github.com/microsoft/PowerToys/issues/19124).
 - There are reports of users who are [unable to open the Settings window](https://github.com/microsoft/PowerToys/issues/18015). This is being caused by incompatibilities with some applications (RTSS RivaTuner Statistics Server and MSI AfterBurner are known examples of this). If you're affected by this, please check the  linked issue to verify if any of the presented solutions works for you.
 
 ### General
 
-- Added a new utility: Screen Ruler.
-- Added a new utility: Quick Accent. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added a new utility: Text Extractor. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
-- Upgraded the Windows App SDK runtimes to 1.1.4.
+- Fixed an issue that caused bug report generation to fail. (This was a hotfix for 0.62)
+- Updated the Windows App SDK runtimes to 1.1.5.
 
 ### Always on Top
 
-- Fixed a bug causing the border to linger when closing an Outlook popup window.
+- Fixed an issue causing the border do linger when moving a window between virtual desktops.
+- The minimum thickness for the borders is now 1. Thanks [@unuing](https://github.com/unuing)!
+- Borders were showing in Virtual Desktop thumbnails. These were removed.
+- Corrected the borders visuals to more closely follow the application borders.
```

Comment:
```suggestion
- Corrected the borders visuals to more closely follow the application borders. Thanks [@SeraphimaZykova](https://github.com/SeraphimaZykova)!
```
This is about the _corners_, right?

Reply:
Yes, but Seraphima is part of the core team and not a community member. We usually call out to community members :)